WORKSPACE

```diff
@@ -21,9 +21,9 @@ switched_rules_by_language(
 http_archive(
     name = "com_github_grpc_grpc",
     urls = [
-        "https://github.com/grpc/grpc/archive/v1.31.0.tar.gz",
+        "https://github.com/grpc/grpc/archive/v1.37.0.tar.gz",
     ],
-    strip_prefix = "grpc-1.31.0",
+    strip_prefix = "grpc-1.37.0",
```

Comment:
@donnadionne Can you check if this is enough?

Reply:
It looks like the file isn't using the SHA256 to verify the contents of the archives, so that ought to be enough for bumping revision from 1.31 to 1.37.